openfisca_nsw_safeguard/variables/ESS_PDRS_Estimator/HVAC1/certificate_estimation/HVAC1_ESC_variables.py

```diff
@@ -312,3 +312,159 @@ class HVAC1_PDRS__postcode(Variable):
+        product_class = buildings('HVAC1_Air_Conditioner_type', period)
+        # AC_Class = (product_class.possible_values)
+        new_AC_cooling_capacity = buildings('HVAC1_cooling_capacity_input', period)
+        cooling_capacity = np.select(
+                                    [
+                                        (new_AC_cooling_capacity < 4),
+                                        ((new_AC_cooling_capacity >= 4) * (new_AC_cooling_capacity < 6)),
+                                        ((new_AC_cooling_capacity >= 6) * (new_AC_cooling_capacity < 10)),
+                                        ((new_AC_cooling_capacity >= 10) * (new_AC_cooling_capacity < 13)),
+                                        ((new_AC_cooling_capacity >= 13) * (new_AC_cooling_capacity < 25)),
+                                        ((new_AC_cooling_capacity >= 25) * (new_AC_cooling_capacity <= 65)),
+                                        (new_AC_cooling_capacity > 65)
+                                    ],
+                                    [
+                                        "less_than_4kW",
+                                        "4kW_to_6kW",
+                                        "6kW_to_10kW",
+                                        "10kW_to_13kW",
+                                        "13kW_to_25kW",
+                                        "25kW_to_65kW",
+                                        "over_65kW"
+                                    ]
+                                    )
+        TCSPF_is_zero = ((AC_TCSPF == 0) + (AC_TCSPF == None))
+        AC_exceeds_benchmark = np.where(
+            TCSPF_is_zero,
+            (AC_AEER >= parameters(period).PDRS.AC.table_D16_5[product_class][cooling_capacity]),
+            (AC_TCSPF >= parameters(period).PDRS.AC.table_D16_4[product_class][cooling_capacity])
+            )
+        return AC_exceeds_benchmark
+    
+    
+
+class HVAC1_HSPF_or_ACOP_exceeds_ESS_benchmark(Variable):
+    """ This variable is used if the AC climate zone is hot or average and there is a GEMS heating capacity
+    """
+    value_type = bool
+    entity = Building
+    definition_period = ETERNITY
+    label = 'Does the Air Conditioner have a HSPF mixed equal or greater than the minimum' \
+            ' HSPF mixed listed in Table D16.3? If the HSPF is not available, is the Rated' \
+            ' ACOP equal or greater than the Minimum Rated ACOP listed in Table D16.5?'
+    metadata = {
+        'alias':  'ESS - HSPF or ACOP exceeds benchmark'
+    }
+
+    def formula(buildings, period, parameters):
+        AC_HSPF_mixed = buildings('HVAC1_HSPF_mixed', period)
+        AC_HSPF_cold = buildings('HVAC1_HSPF_cold', period)
+        AC_ACOP = buildings('HVAC1_rated_ACOP_input', period)
+        product_class = buildings('HVAC1_Air_Conditioner_type', period)
+        new_AC_heating_capacity = buildings('HVAC1_heating_capacity_input', period)
```

Comment:
This should be for cooling capacity (not heating capacity), as the rule tables refer only to cooling capacity.